sc.it.homework.Concurrency/src/main/java/Concurrency/FestivalStatisticsThread.java

```diff
@@ -0,0 +1,36 @@
+package Concurrency;
+import java.util.Collections;
+import java.util.LinkedList;
+public class FestivalStatisticsThread implements Runnable {
+    String threadName;
+    static FestivalGate gate = new FestivalGate();
+    LinkedList list;
+    int fullTickets = 0;
+    int freePass = 0;
+    int vipPass = 0;
+    int oneDayPass = 0;
+    int oneDayVIP = 0;
+    public FestivalStatisticsThread(String name) {
+        threadName = name;
+        list = gate.generateAttendeeThread();

```

Comment:
the logic here is a bit circular. the statistics shouldn't generate the attendees, the list is blocked when creating a new one, stats are not computed when expected, rethink the logic a bit